packages/components/src/local/mapping/index.stories.tsx

```diff
@@ -243,3 +244,44 @@ WithPhases.story = {
     }
   }
 }
+
+/**
+ * VIEW WITH ASSET ROUTES
+ */
+const trimmedLabel = 'Trimmed'
+const trimmedDefaultValue = false
+const selectedLabel = 'Selected'
+const selectedDefaultValue = false
+
+
+const greenForce: any = forces[3]
+const platform: any = greenForce.assets[0]
+const planned = platform.plannedTurns
+const history = platform.history
+
+export const WithRoute: React.FC = () => <Mapping
+  bounds = {bounds}
+  tileLayer = {LocalTileLayer}
+  tileDiameterMins={5}
+  forces={forces}
+  phase = {Phase.Planning}
+  playerForce='Green'>
```

Comment:
```suggestion
  playerForce='Green'
```
This is a minor bug that has been introduced in #393 but if you merge in `feature/381-mapping-refactor` once #391 has been merged it will fix this.

Reply:
got it, thanks.